common/buildings/00_tradepost.txt

```diff
@@ -463,3 +463,294 @@ trade_post = {
 		# extra_tech_building_start = 10.0 # Never get at start of game
 	# }
 }
+
+##########################################
+#
+# Ciphers Trade Posts mod merger buildings
+#
+##########################################
+
+trade_post = {	
+	
+#Garrison
+
+	#StableMaster
+	wc_stableMaster = {
```

Comment:
This building should not be in non-trade route posts. `trade_route_wealth` isn't used in republic trade posts. `local_tax_modifier` may give bonus to provinces you don't own.